containers/daap-get-glue-metadata/tests/unit/conftest.py

```diff
@@ -0,0 +1,41 @@
+import sys
+import os
+from os.path import dirname, join
+from dataclasses import dataclass
+
+from moto import mock_glue
+import boto3
+import pytest
+
+sys.path.append(join(dirname(__file__), '../', '../', 'src', 'var', 'task'))
```

Comment:
This path hacking is necessary because our source is organised according to the container layout, rather than as a python package with __init__.py files.

We also have the shared libraries which are distributed in the base layer for the images.